plugins/usd/f3d-usd-formats.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mime-info xmlns="http://www.freedesktop.org/standards/shared-mime-info">
+  <mime-type type="model/vnd.usd">
```

Comment:
not coherent with what is declared in the CMakeLists.txt. Only what is not available in IANA should be listed in this file.

Reply:
I'm confused. What should I do?  
See https://github.com/f3d-app/f3d/pull/819#discussion_r1218291379

Reply:
these .xml files are used to augment the mime types database on linux. If a format is already declared in IANA, there is great chances that it is also already present in the freedesktop database of mime types, which can easily be checked locally.

If a format is declared in IANA but not present in the freedesktop database, then it should be declared here with the right IANA name, it could also be contributed back to freedesktop upstream if we want to.

If a format is not declared in IANA (and objvously not present in the freedesktop database), then it should be declared here  with the `application/vnd.ext` style.

Once a format is correctly declared in the mimetypes database, it can be used by the thumbnailers, if the mime-type are correct.


In your specific case, afaiu:
 - IANA describes `model/vnd.usda` and `model/vnd.usdz`
 - Freedesktop doesnt have any of these mimetypes
 
 So what you need to do:
  - In this .xml file, declare `model/vnd.usda` and `model/vnd.usdz`
  - In this .xml file, declare `application/vnd.usd` and `application/vnd.usdc`
  - in the thumbnail file, use the right mimetype described in the xml file
  - Optionally, contribute back `model/vnd.usda` and `model/vnd.usdz` to freedesktop

Reply:
I see, thank you

Reply:
I'd say yes but I'm not enterely sure. Just check the thumbnailers works locally ?